src/Mvc/test/WebSites/BasicWebSite/RazorComponents/FetchData.razor

```diff
@@ -1,6 +1,6 @@
 ﻿@using BasicWebSite.Services
 @inject WeatherForecastService ForecastService
-
+@preservewhitespace true
```

Comment:
Intentional?

Reply:
Yes, it means I don't have to change the test case in a weird way (which wouldn't really be beneficial since the MVC test case isn't meant to be about how the component does its rendering internally).